### ISSUE TITLE
Map thumbnail shows all/most data locations. Big map does not.

### ISSUE THREAD

**Issue by njarboe** [earthref/MagIC#520]
earthref.org/MagIC/16972
![Screen Shot 2020-11-02 at 4 35 22 PM](https://user-images.githubusercontent.com/1278789/97934251-05ea3880-1d2a-11eb-821f-4948a5df3303.png)
![Screen Shot 2020-11-02 at 4 35 30 PM](https://user-images.githubusercontent.com/1278789/97934262-0b478300-1d2a-11eb-84cc-21e1521bf112.png)

**Comment by rminnett** [comment]
Ok, thanks - Anthony had also suggested just displaying the sites lat/lon and not using the location bounding box unless that's all we have.

**Comment by njarboe** [comment]
I agree. Just site locations give better information when available.
